package.json

```diff
@@ -34,12 +34,17 @@
   "author": "HomeOffice",
   "dependencies": {
     "browserify": "^12.0.1",
+    "express": "^4.14.0",
     "hof-bootstrap": "^6.0.0",
```

Comment:
There are none. There are only adding dependencies that were previously undefined, I assume to satisfy the eslint rules.